### PR TITLE
docs(common): add trackBy implementation examples

### PR DESCRIPTION
**Pull request by DzmitryShylovich**

### CODE REVIEW

File: modules/@angular/common/src/directives/ng_for_of.ts [hunk 1]
```diff
@@ -65,21 +68,38 @@ export class NgForOfRow<T> {
...
  *
  * ```
  * <ng-template ngFor let-item [ngForOf]="items" let-i="index" [ngForTrackBy]="trackByFn">
  *   <li>...</li>
  * </ng-template>
- * ```
- *
- * ### Example
  *
- * See a [live demo](http://plnkr.co/edit/KVuXxDp0qinGDyo307QW?p=preview) for a more detailed
```
Comment (DzmitryShylovich) [inline modules/@angular/common/src/directives/ng_for_of.ts]: the plunkr is outdated

File: modules/@angular/common/src/directives/ng_for_of.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {ChangeDetectorRef, Directive, DoCheck, EmbeddedViewRef, Input, IterableChangeRecord, IterableChanges, IterableDiffer, IterableDiffers, NgIterable, OnChanges, SimpleChanges, TemplateRef, TrackByFunction, ViewContainerRef, forwardRef, isDevMode} from '@angular/core';
```
Comment (Toxicable) [inline modules/@angular/common/src/directives/ng_for_of.ts]: should be in separate PR
